test/libsolidity/smtCheckerTests/natspec/natspec_smtchecker_empty.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+	/// @custom:smtchecker
+	function f(uint x) internal pure returns (uint) {
+		return x;
+	}
+}
+// ----
+// Warning 3130: (38-102): Unknown option for "custom:smtchecker": ""
```

Comment:
Would you get the same error if SMT checker is disabled?

Reply:
Nop